prog/probe.go

```diff
@@ -236,7 +234,7 @@ func probeMain(flags probeFlags, targets []appclient.Target) {
 
 	if flags.weaveEnabled {
 		client := weave.NewClient(sanitize.URL("http://", 6784, "")(flags.weaveAddr))
-		weave, err := overlay.NewWeave(hostID, client, dockerEndpoint)
+		weave, err := overlay.NewWeave(hostID, client, "")
```

Comment:
Good point, I was thinking to keep `overlay.NewWeave` backwards compatible but I presume there is no need.. will remove it!